MNE/disp3D/3DObjects/bem/bemsurfacetreeitem.h

```diff
@@ -169,6 +169,30 @@ private slots:
     */
```

Comment:
don't use slots anymore, just connect to member methods. so just call it private: - exception, when its inherited from base class